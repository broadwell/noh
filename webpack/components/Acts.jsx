import React from "react";

const Acts = () => (
  <div className="acts-container">
    <div className="tl__act" style={{ width: "35%" }}>
      First act (mae ba)
    </div>
    <div className="tl__act" style={{ width: "65%" }}>
      Second act (nochi ba)
    </div>
  </div>
);

export default Acts;
