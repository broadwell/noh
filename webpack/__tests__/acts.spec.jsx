import React from "react";
import { shallow } from "enzyme";
import Acts from "../components/Acts";

describe("<Acts>", () => {
  it("renders as expected by default", () => {
    const component = shallow(<Acts />);
    expect(component).toMatchSnapshot();
  });
});
