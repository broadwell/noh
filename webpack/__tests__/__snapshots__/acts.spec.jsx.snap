// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Acts> renders as expected by default 1`] = `
<div
  className="acts-container"
>
  <div
    className="act"
    key="act-Musicians-8.157715839564922"
    style={
      Object {
        "width": "8.157715839564922%",
      }
    }
  >
     
    <span
      className="translation"
    >
      (
      Musicians
      )
    </span>
  </div>
  <div
    className="act"
    key="act-First Act-29.29979605710401"
    style={
      Object {
        "width": "29.29979605710401%",
      }
    }
  >
    <span
      className="transcription"
    >
      Mae bae
    </span>
     
    <span
      className="translation"
    >
      (
      First Act
      )
    </span>
  </div>
  <div
    className="act"
    key="act-Second Act-54.384772263766145"
    style={
      Object {
        "width": "54.384772263766145%",
      }
    }
  >
    <span
      className="transcription"
    >
      Nochi ba
    </span>
     
    <span
      className="translation"
    >
      (
      Second Act
      )
    </span>
  </div>
  <div
    className="act"
    key="act--8.157715839564922"
    style={
      Object {
        "width": "8.157715839564922%",
      }
    }
  >
    <span
      className="transcription"
    >
      Oshirabe
    </span>
     
  </div>
</div>
`;
